src/Microsoft.ML.Data/DataView/LambdaColumnMapper.cs

```diff
@@ -140,7 +140,7 @@ private sealed class Impl<T1, T2, T3> : OneToOneTransformBase
                 Metadata.Seal();
             }
 
-            public override void Save(ModelSaveContext ctx)
+            private protected override void SaveModel(ModelSaveContext ctx)
```

Comment:
>SaveModel [](start = 44, length = 9)

May we call it `SaveCore` following the convention of `TrainCore`?

Reply:
There are quite a few instances where there is a `SaveCore` method that was referenced by `SaveModel` (the previous `Save` method). That's why I preferred not to name it `SaveCore`. If it's fine with you I would keep it like this.